third_party/llvm/expand_isl_cmake_vars.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
```

Comment:
How is this different from tensorflow/third_party/llvm/expand_cmake_vars.py ?

Reply:
The `_CMAKE_VAR_REGEX` for parsing isl source files is not the same as the one for LLVM.

```
$ diff expand_cmake_vars.py expand_isl_cmake_vars.py
27c27
< _CMAKE_VAR_REGEX = re.compile(r"\${([A-Za-z_0-9]*)}")
---
> _CMAKE_VAR_REGEX = re.compile(r"@([A-Za-z_0-9]*)@")
```

Reply:
In that case can you please extract out a helper function to a common .py file and call that from both `expand_cmake_vars.py` and this one, passing `_CMAKE_VAR_REGEX` as an argument?  This duplicated code will otherwise become a maintenance issue.

I'd also be fine with a "master" `expand_cmake_vars.py` that takes `_CMAKE_VAR_REGEX` and as a command line argument (which will be passed in via the bazel rule).

Why is `_CMAKE_VAR_REGEX` different here btw?  Can we do something to make them the same?

+CC @hawkinsp just in case I'm misleading you